src/main/java/it/pagopa/pm/gateway/controller/CreditCardPaymentController.java

```diff
@@ -52,6 +45,9 @@ public class CreditCardPaymentController {
     @Autowired
     private VposDeleteService deleteService;
 
+    @Value("${vpos.polling.url}")
+    private String vposPollingUrl;
```

Comment:
Can you refactor this class to use constructor injection?

Reply:
https://github.com/pagopa/pagopa-payment-transactions-gateway/pull/206/commits/406cf3b4aedf233f0651a6f2405ea0fb94ccf3db